js/main.js

```diff
@@ -155,12 +155,27 @@ function openingRangeContainsTime(openingRange, date) {
 
 /*
  * Returns opening ranges compiled via opening_hours.js.
+ * Returns the next opening date if no opening ranges are available.
+ * Returns null if not.
  */
 function getOpeningRanges(openingHoursStrings) {
     var monday = moment().startOf("week").add(1, 'days').toDate();
     var sunday = moment().endOf("week").add(1, 'days').toDate();
     var oh = new opening_hours(openingHoursStrings);
-    return oh.getOpenIntervals(monday, sunday);
+    var intervals = oh.getOpenIntervals(monday, sunday);
+    var nextChange = oh.getNextChange();
+
+    if(intervals.length > 0) {
```

Comment:
Please stick to the formatting as being used in the file such as `if (` instead of `if(`.